webviz_subsurface/plugins/_inplace_volumes.py

```diff
@@ -101,6 +101,11 @@ def __init__(
         self.uid = uuid4()
         self.selectors_id = {x: str(uuid4()) for x in self.selectors}
         self.plotly_theme = app.webviz_settings["theme"].plotly_theme
+        self.initial_plot = "Per realization"
+        self.initial_group = None
+        if len(self.volumes["ENSEMBLE"].unique()) > 1:
```

Comment:
Maybe if else, rather than first setting the variables, and then overwriting them in the very likely case that there is more than one ensemble?